backend/data/models.py

```diff
@@ -69,76 +70,131 @@ class PersonName(Field, models.Model):
         to=Person,
         on_delete=models.CASCADE,
         null=False,
-        related_name='names',
+        related_name="names",
         unique=True,
     )
 
     def __str__(self):
         return self.value
 
 
+class CaseStudy(models.Model):
+    """
+    A case study is an overarching collection of epistolary events, bound together by a common theme, e.g. `The Saga of St. Boniface` or `The  Nun Rebellion of Poitiers`.
+    """
+
+    class Meta:
+        verbose_name = "case study"
+        verbose_name_plural = "case studies"
+
+    name = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+    )
+
+    def __str__(self):
+        return self.name
+
+
 class EpistolaryEvent(models.Model):
+    """
+    Epistolary events are groups of related letter actions that are connected in some way.
+
+    For instance, a political campaign (epistolary event) may consist of the writing, transporting and reading of individual letters (letter actions).
+    """
+
+    name = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+    )
+
+    case_studies = models.ManyToManyField(
+        to=CaseStudy,
+        related_name="epistolary_events",
+        help_text="case studies this event belongs to",
+    )
+
+    note = models.TextField(
+        null=False,
+        blank=True,
+        help_text="Additional notes that describe the event and what connects the letter actions it comprises."
+    )
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class LetterAction(models.Model):
+    """
+    A letter action is an atomic action performed on a letter, e.g. writing, delivering, reading.
+
+    These can be grouped into epistolary events.
+    """
+
     letters = models.ManyToManyField(
```

Comment:
A `LetterAction` currently contains multiple letters; it inherited this from the old `EpistolaryEvent` model. What would you say if we made this a 1-to-1 relationship? That would fit the idea of an 'atomic' action more.

Reply:
I would agree that "atomic" implies it's one-to-one, but maybe this doesn't make sense for the researchers. If a messenger delivers three letters at once, you could consider that an "atomic action" as it can't be broken up into a sequence of actions.

I'd keep it like this for now, and check in in our next meeting.